packages/concerto-types/scripts/codegen.js

```diff
@@ -4,12 +4,14 @@ const { ModelManager } = require('@accordproject/concerto-core');
 const { CodeGen: { TypescriptVisitor }} = require('@accordproject/concerto-codegen');
 const { FileWriter } = require('@accordproject/concerto-util');
 const path = require('path');
+const { DCS_MODEL }= require('@accordproject/concerto-core');
```

Comment:
You'll still have to export it via `index.js`. I don't think this codegen would work without it.